samples/sensor/veml7700/src/main.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022, Nikolaus Huber
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+#include <stdio.h>
+#include <sys/__assert.h>
+
+int main(void)
+{
+	const struct device *veml7700 = DEVICE_DT_GET_ANY(vishay_veml7700);
+
+	__ASSERT(veml7700 != NULL, "Failed to get device binding");
+	__ASSERT(device_is_ready(veml7700), "Device %s is not ready",
+			veml7700->name);
```

Comment:
assertions may not always be evaluated, to make sample more user-friendly better print errors and return.